PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -227,12 +227,19 @@ AliAnalysisTaskGammaConvV1::AliAnalysisTaskGammaConvV1(): AliAnalysisTaskSE(),
   fHistoSPDClusterTrackletBackground(NULL),
   fHistoV0MultVsNumberTPCoutTracks(NULL),
   fHistoNV0Tracks(NULL),
+  fHistoBDToutput(NULL),
+  fHistoBDToutputPt(NULL),
+  fHistoBDToutputMCTrue(NULL),
   fProfileEtaShift(NULL),
   fProfileJetJetXSection(NULL),
   fhJetJetNTrials(NULL),
   fHistoEtaShift(NULL),
   tESDMesonsInvMassPtDcazMinDcazMaxFlag(NULL),
   fInvMass(0),
+  fBDTvariable(NULL),
```

Comment:
There are warnings related to this PR during compilation, could you please fix them?